core/src/main/resources/error/error-classes.json

```diff
@@ -656,6 +656,11 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_EMPTY_LOCATION" : {
+    "message" : [
+      "The location name cannot be empty string or null, but `<location>` was given."
```

Comment:
Could you add a test when the location is NULL. I just worry that parameters substitution might not handle this. Please, check this.

Reply:
Good catch.
I tried to add a test with NULL, but it complains [PARSE_SYNTAX_ERROR] rather than [INVALID_EMPTY_LOCATION].
Let me exclude the null from the error message.